DashKit.xcodeproj/project.pbxproj

```diff
@@ -699,8 +702,8 @@
 			inputFileListPaths = (
 			);
 			inputPaths = (
-				"$(BUILT_PRODUCTS_DIR)/DashKit.framework",
-				"$(BUILT_PRODUCTS_DIR)/DashKitUI.framework",
+				"$(TARGET_BUILD_DIR)/DashKit.framework",
```

Comment:
[nit] Intended? Do we need to make this change for the other plugins?

Reply:
Yes, intended. This is fixing an issue when building an archive, that the files in BUILT_PRODUCTS_DIR are symlinks to TARGET_BUILD_DIR.  I chose to do this here, since we're not using the copy-frameworks.sh file, which I've modified to pre-follow the symlinks and pass modified vars to carthage.  Other plugins should either do this change or use the copy-frameworks.sh script.